templates/pwm.asg.element.json

```diff
@@ -0,0 +1,884 @@
+                                "command": {
+                                    "Fn::Join": [
+                                        "",
+                                        [
+                                            "git clone https://github.com/",
+                                            {
+                                                "Ref": "GitHubUsername"
+                                            },
+                                            "/",
+                                            {
+                                                "Ref": "GitHubRepo"
+                                            },
+                                            " /tmp/",
+                                            {
+                                                "Ref": "GitHubRepo"
+                                            },
+                                            "\n"
+                                        ]
+                                    ]
+                                }
+                            },
+                            "41-mvpwmformula": {
+                                "command": {
+                                    "Fn::Join": [
+                                        "",
+                                        [
+                                            "cp -r /tmp/",
+                                            {
+                                                "Ref": "GitHubRepo"
+                                            },
+                                            "/* /srv/salt/",
+                                            "\n"
+                                        ]
+                                    ]
+                                }
+                            },
+                            "43-getformulapillar": {
+                                "command": {
+                                    "Fn::Join": [
+                                        "",
+                                        [
+                                            "aws s3 cp s3://",
+                                            {
+                                                "Ref": "ConfigBucketName"
+                                            },
+                                            "/",
+                                            {
+                                                "Ref": "SaltPillarFile"
+                                            },
+                                            " /srv/salt/pillar/",
+                                            {
+                                                "Ref": "SaltPillarFile"
+                                            },
+                                            "\n"
+                                        ]
+                                    ]
+                                }
+                            },
+                            "80-firewalld": {
```

Comment:
Is this OBE? Isn't salt managing the firewalld config?

Reply:
Ergh. Well salt _should_ be managing firewalld, https://docs.saltstack.com/en/latest/ref/states/all/salt.states.firewalld.html

Reply:
We can leave this for now, pin for later